roles/rabbitmq/templates/etc/serverspec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'socket'
+
+describe user('rabbitmq') do
+    it { should exist }
+    it { should belong_to_group 'rabbitmq' }
+end  
+
+describe file('/etc/nova/nova.conf') do
+  it { should_not contain /(rabbit_password=\w{0,7})$/ }
```

Comment:
I'm guessing this is a check to see if the rabbit password is long enough. That's fine, this will just break if we ever move rabbit to its own host.